handlers/iter8.go

```diff
@@ -49,6 +49,19 @@ func Iter8ExperimentGet(w http.ResponseWriter, r *http.Request) {
 		handleErrorResponse(w, err)
 		return
 	}
+	workloadList, err := business.Workload.GetWorkloadList(namespace)
+	if err != nil {
+		handleErrorResponse(w, err)
+		return
+	}
+	workloads := workloadList.Workloads
+	for _, w := range workloads {
+		if w.Name == experiment.ExperimentItem.Baseline {
+			experiment.ExperimentItem.BaselineVersion = w.Labels["version"]
```

Comment:
Note, Istio can change this label, so in Kiali code we use the config to fetch this value:

```
version := workload.Labels[conf.IstioLabels.VersionLabelName]
```